app/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "daydream-scope-desktop",
+  "productName": "Daydream Scope",
+  "version": "0.1.0-alpha.7",
```

Comment:
I noticed that the PR description noted that the Electron app is versioned separately from the Python package.

Could you elaborate on the rationale for this? I would've imagined that it'd be simpler to use the same version for everything since we already do so for the frontend and the Python project. But, curious if you see a benefit in specifically versioning the Electron app differently.

Reply:
Discussed offline to switch everything to proper sem ver.